### ISSUE TITLE
SUMMARY

### ISSUE THREAD

**Issue by r-barnes** [holukas/dyco#13]
I feel that reading the summary at https://github.com/holukas/dyco/blob/main/paper/paper.md does not give me a clear idea of what DYCO does. The summary mixes together background information and summary about the program.

I feel it would be clearer to state the background info in one paragraph and then explain in summary how DYCO solves the problem.

openjournals/joss-reviews/issues/2575

**Comment by holukas** [comment]
I restructured the paper and re-worded the _Summary_, which now contains some of the info previously found in _Scientific background_ in the first paragraph and then mentions how DYCO can help in addressing the challenges (61ee45db749239be03bcadb8f2f7aa4d6e9fc123).

The _Statement of need_ was also reworded and now goes into more detail regarding the unique features DYCO can offer to solve commonly encountered problems when dealing with low-signal-to-noise-ratio compounds.

In addition, I felt it would be helpful to give the reader a quick overview of the _Processing chain_ in a dedicated section. 

I also updated the _Real-world examples_ with a short example how DYCO can help with the calculation of N2O fluxes over a managed grassland.
